src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+
+  if (days >= 7) {
+    return dayRent * days - 50;
```

Comment:
Replace the hard-coded value '50' with a constant that explains what this value represents, as per the checklist item 2. You can name it something like `LONG_TERM_DISCOUNT` to make the code more readable and maintainable.